<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>311</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="findButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Искать FT232</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>490</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить дисплей</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>460</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SEThButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ВЫСОКИЙ УРОВЕНЬ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SETlButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>НИЗКИЙ УРОВЕНЬ</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
